### PR TITLE
Make code optional

### PR DESCRIPTION
**Pull request by cedvan**
`code` is displayed only if there are greater than `0`

### CODE REVIEW

**Comment by mojoLyon** [comment]
:metal:

File: Resources/doc/configuration.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Node to configure bundle `m6web_api_exception`
         - `...`
     - `...`
     
-*NB : `status` and `headers` are enabled only for exceptions that implements `M6Web\Bundle\ApiExceptionBundle\Exception\Interfaces\HttpExceptionInterface`*
+*NB : `code` is displayed only if there are greater than `0`*
```
Comment (mikaelrandy) [inline Resources/doc/configuration.md]: "[...] is [...] there are [...]" strange formulation :/

File: Resources/doc/usage.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ m6web_api_exception:
         exceptions:
             Acme\DemoBundle\Exception\UserNotFoundException:
                 status: 404 # Used for exceptions implements HttpExceptionInterface
-                code: 5286 # Create an unique code for this exception in your API
+                code: 5286 # Create an unique code for this exception in your API, optional default to 0 and not displayed
```
Comment (mikaelrandy) [inline Resources/doc/usage.md]: "optional, default to 0 and not displayed"

**Comment by fabdsp** [comment]
:+1:

File: Resources/doc/configuration.md (hunk 1 above)
Comment (cedvan) [inline Resources/doc/configuration.md]: Give me must formulation please :)

Comment (mikaelrandy) [inline Resources/doc/configuration.md]: > `code` is displayed only if greater than `0`

File: Resources/doc/usage.md (hunk 2 above)
Comment (cedvan) [inline Resources/doc/usage.md]: ?

Comment (mikaelrandy) [inline Resources/doc/usage.md]: note the coma between "optional" and "default"

Comment (cedvan) [inline Resources/doc/usage.md]: ok :)

Comment (cedvan) [inline Resources/doc/usage.md]: done

File: Resources/doc/configuration.md (hunk 1 above)
Comment (cedvan) [inline Resources/doc/configuration.md]: done

**Comment by cedvan** [comment]
New release [v1.0.1](https://github.com/M6Web/ApiExceptionBundle/releases/tag/v1.0.0)

**Pull request merged**